🚀 Add Brook & Sam's Talks (#6)
JackDevAU/ssw-brisbane-knowledge-share: src/content/knowledge/building-a-terminal-file-explorer-with-zig.md

---

> title: "Building a Terminal File Explorer: Inside the 'Jido' Project"

Author: ```suggestion
title: "Building a Terminal File Explorer with Zig"
```
